modules/programs/git.nix

```
@@ -239,6 +239,32 @@ in {
         };
       };
 
+      difftastic = {
+        enable = mkEnableOption "" // {
+          description = ''
+            Enable the <command>difft</command> syntax highlighter.
+            See <link xlink:href="https://github.com/Wilfred/difftastic" />.
+          '';
+        };
+        background = mkOption {
+          type = types.enum [ "light" "dark" ];
+          default = "light";
+          example = "dark";
+          description = ''
+            Options to configure if difftastic should use the lighter or darker
+            colors for syntax highlithing.
+          '';
+        };
+        color = mkOption {
+          type = types.enum [ "always" "auto" "never" ];
+          default = "auto";
+          example = "always";
+          description = ''
+            Options to configure if difftastic should colors in it's output.
```

Perhaps something like
```suggestion
            Determines when difftastic should color its output.
```